devicesetup/src/main/res/values/strings.xml

```diff
@@ -61,6 +61,7 @@
     <string name="verify_product_ownership">Verifying product ownership</string>
     <string name="action_done">Done</string>
     <string name="title_activity_password_reset">Password Reset</string>
+    <string name="title_activity_two_factor">Two Factor Authentication</string>
```

Comment:
@channel — something we decided in our stand up today around communication:

- In _internal_ and _technical_ communications, we will refer to this feature as *multi-factor authentication (MFA)*
- In _external_ and _customer-facing_ communications, we will refer to this feature as *two-step authentication*